### PR TITLE
update answers, insert new Q&A

### PR DESCRIPTION
**Pull request by katelynsyu**
Removed FAQ answers that no longer pertain to HOTH 9 (ie. mentorship application form). 
Added some new questions and answers:
- date and duration of HOTH 9
- location (online or offline) - might have to be fixed based on how the school proceeds with Covid and if ResLife ever gets back with an in-person location to host the hackathon
- plugged the discord

### CODE REVIEW

File: src/components/HomePage/FAQSection.js
Comment (khxia) [inline src/components/HomePage/FAQSection.js:134]: Our current FAQ uses a combination of `'` and `’`. This is probably the fault of the people who made the website in the previous year. However we want to be as consistent as possible. Can you change all instances of  `’` to `'`?

You may not need to use the `&apos` in the template strings, just a `'` should be enough.

File: src/components/HomePage/FAQSection.js
Comment (khxia) [inline src/components/HomePage/FAQSection.js:151]: `Discord server`
And have the link be for both words.

File: src/components/HomePage/FAQSection.js
Comment (khxia) [inline src/components/HomePage/FAQSection.js:176]: Did christeinar confirm that we'll be using the Hack email for mentor applications? I would have thought that the discord server would have been more approachable.

File: src/components/HomePage/FAQSection.js
Comment (khxia) [inline src/components/HomePage/FAQSection.js:183]: Same here for the discord link. 
Also make sure that the discord link that you created is a permanent invite link.

**Review by khxia — CHANGES_REQUESTED** [review]

**Review by khxia — APPROVED** [review]
LGTM. Good job!

Feel free to merge this after Einar has had a look at it and approves.

File: src/components/HomePage/FAQSection.js
Comment (EinarBalan) [inline src/components/HomePage/FAQSection.js:126]:
```suggestion
		answer: `Applications for HOTH 9 are open to anyone! However, we are 
		prioritizing UCLA students for our prize tracks as well as in person hacking.`
```

File: src/components/HomePage/FAQSection.js
Comment (EinarBalan) [inline src/components/HomePage/FAQSection.js:167]: nit: roadblocks to road blocks

File: src/components/HomePage/FAQSection.js
Comment (EinarBalan) [reply src/components/HomePage/FAQSection.js:176]: agreed, we should add a link to the discord here

**Review by EinarBalan — CHANGES_REQUESTED** [review]
Left some comments, but other than that everything looks great! After you push your changes, we'll take one last look and it should be good to merge.

**Review by EinarBalan — APPROVED** [review]
Looks great!

**Pull request merged**
